src/Administration/Resources/app/administration/src/module/sw-product/page/sw-product-detail/index.js

```diff
@@ -460,7 +460,11 @@ Component.register('sw-product-detail', {
                             seoUrl.isModified = true;
                         }
 
-                        updatePromises.push(this.seoUrlService.updateCanonicalUrl(seoUrl, seoUrl.languageId));
+                        if (seoUrl.isNew()) {
```

Comment:
Some sort of e2e-test would be awesome :-)

Reply:
I'm not sure if I can provide them in a reasonable time. I'll have a look if I find someone else to write some.